solr/benchmark/src/resources/configs/cloud-minimal/conf/schema.xml

```diff
@@ -44,11 +44,16 @@
     <dynamicField name="*_b" type="boolean" indexed="true" stored="true"/>
     <dynamicField name="*_s" type="string" indexed="true" stored="false"/>
     <dynamicField name="*_t" type="text" indexed="true" stored="false"/>
-    <dynamicField name="*_i" type="int" indexed="false" stored="false"/>
```

Comment:
Have you run the full test suite to make sure this is compatible with tests that use `cloud-minimal`?

Reply:
Good call, I thought about it when I changed the schema but I forgot after. 
I've updated `JsonFaceting` test to use the dv + indexed field (used to be dv only). Should be no difference. `CloudIndexing` was using the non-indexed fields which is strange to me, maybe the schema was updated after the test or maybe the test was focusing just on the distribution? I left it using the indexed fields now, which is more realistic, if someone is trying to test something in particular with non-indexed fields they can update again.